sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-ministry-student-unmet-need-report.sql

```diff
@@ -0,0 +1,86 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Ministry_Student_Unmet_Need_Report',
+    'SELECT
+      users.first_name AS "Student First Name",
+      users.last_name AS "Student Last Name",
+      sin_validations.sin AS "SIN",
+      applications.student_number AS "Student Number",
+      users.email AS "Student Email Address",
+      students.contact_info ->> ''phone'' AS "Student Phone Number",
+      institution_locations.institution_code AS "Institution Location Code",
+      institution_locations.name AS "Institution Location Name",
+      applications.application_number AS "Application Number",
+      CAST(
+        CAST(student_assessments.assessment_date AS date) AS varchar
+      ) AS "Assessment Date",
```

Comment:
You don't need to do a double CAST, do you?

Doing a cast to date is enough:
`select CAST(assessment_date AS date) AS "Assessment Date" from sims.student_assessments`

Reply:
Nope, that alone doesn't work. `timestamptz` requires CAST to `date` and then to `varchar`.

Reply:
It is following what was done for the other reports. Differently from the entity models, when the raw query is executed, the result from a date will contain the time also, which we would like to avoid for this scenario, unless something changes from the previous PRs where the same situation happened.

Reply:
oh, I see you do that to format to `YYYY-MM-DD`.

Reply:
Shouldn't all dates be in the format `YYYY-MM-DD` ?
![image](https://github.com/bcgov/SIMS/assets/78114138/d1ce38f7-b9d0-43f7-bdec-4ddbd0369bc3)
![image](https://github.com/bcgov/SIMS/assets/78114138/2879ed06-b25d-4006-aa64-0021c54fd0d2)

Reply:
The generated CSV should have the format `YYYY-MM-DD` for all dates (the first one for the assessment date should be wrong in the requirement). Is it not right while in CSV format?
While opening as a spreadsheet, it will be interpreted as a date, as should be expected. That is my understanding and how the other reports are working right now.

Reply:
The CSV is ok (`YYYY-MM-DD`).  All good then.